src/main/java/gov/nist/oar/rmm/repositories/impl/CustomRepositoryImpl.java

```diff
@@ -48,157 +48,131 @@
 	@Autowired
 	MongoConfig mconfig;
 
-        @Autowired
-        AppConfig appconfig;
-	
-	/* (non-Javadoc)
-	 * @see gov.nist.oar.rmm.repositories.RecordRepository#find()
-	 * Search with given criteria
+	@Autowired
+	AppConfig appconfig;
+
+	/**
+	 * Find the record with given search parameters. Returns JSON document of
+	 * results.
 	 */
 	@Override
-	public Document find(Map<String,String> params){	
+	public Document find(Map<String, String> params) {
+
+//		logger.info("Find record with given parameters:" + params);
```

Comment:
When the client requests a search that results in this function getting called, is the request logged with the parameters anywhere else?  If not, perhaps this line should be retained.